### PR TITLE
2.2.0-10.3 beta

### PR DESCRIPTION
**Pull request by devoncarew**
- regenerate icons and other metadata for framework version `2.2.0-10.3 beta` (done as several commits to try and make a review meaningful)

### CODE REVIEW

**Comment by DanTup** [comment]
Something looks slightly strange with car_fill (like there are two copies of it overlayed?):

Screenshot 2021-05-18 at 07 12 57

The new one looks like an improvement over the original, but I don't know if it's intended to look like this. It looks the same at https://flutter.github.io/cupertino_icons/ so if there's an issue it's probably not with this generation though.

File: resources/catalog/widgets.json [hunk 1]
```diff
@@ -191,6 +191,12 @@
       "library": "widgets",
       "description": "A widget that attempts to size the child to a specific aspect ratio."
     },
+    {
+      "name": "Autocomplete",
+      "parent": "StatelessWidget",
+      "library": "material",
+      "description": "{@macro flutter.widgets.RawAutocomplete.RawAutocomplete}"
```
Comment (DanTup) [inline resources/catalog/widgets.json:198]: Unrelated to this PR, but depending on how this data is intended to be used, would it be better if this was resolved to the underlying comment?

**Review by DanTup — APPROVED** [review]
lgtm with some comments/questions (that probably don't need any changes here).

It looks like tool/icon_generator/lib/material.dart may be sorted differently now which caused a larger diff than probably necessary - there may be value in making it sorted consistently for reduced diffs/simpler PRs.

**Comment by devoncarew** [comment]
> It looks like tool/icon_generator/lib/material.dart may be sorted differently now which caused a larger diff than probably necessary - there may be value in making it sorted consistently for reduced diffs/simpler PRs.

Totally agree - I'll file an issue for this.

> Unrelated to this PR, but depending on how this data is intended to be used, would it be better if this was resolved to the underlying comment?

Yup; currently that's pretty difficult to do - it requires some sophistication, and only the analysis server and dartdoc know how to resolve `{@macro ...}` comments. We could simplify things here; one possibility is to have macro values read from a well know location in a package (`lib/macros.md`, ...). cc @jcollins-g

**Pull request merged**